.github/stale.yml

```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 90
+daysUntilClose: 7
+exemptLabels:
+  - Critical
+  - RFC
```

Comment:
Shouldn't we also leave the `Feature` label?

Reply:
This probot works in two phases:
 - when an issue / PR is not updated for 90 days, it posts a comment and marks it as stale
 - when a stale issue / PR is not updated for another 7 days, it is closed
 - when a stale issue / PR is updated within 7 days, it removes stale label and the whole process starts again

So the reasoning behind it was to close those issues / PRs that people are not longer interested in. If there's still an interest, only four actions in a year would prevent it from closing :)

Reply:
:+1: